light-curve/light_curve/light_curve_py/features/rainbow/temperature.py

```diff
@@ -0,0 +1,176 @@
+from abc import abstractmethod
+from dataclasses import dataclass
+from typing import Dict, List
+
+import numpy as np
+
+__all__ = ["temperature_terms", "BaseTemperatureTerm", "ConstantTemperatureTerm", "SigmoidTemperatureTerm"]
+
+
+@dataclass()
+class BaseTemperatureTerm:
+    """Bolometric term for the Rainbow"""
+
+    @staticmethod
+    @abstractmethod
+    def parameter_names() -> List[str]:
+        return NotImplementedError
+
+    @staticmethod
+    @abstractmethod
+    def parameter_scalings() -> List[str]:
+        """Describes how to unscale the parameters - like time, timescale, flux or do not scale"""
```

Comment:
Could you please list all variants here?

Reply:
done